lib/cli/Cargo.toml

```diff
@@ -107,5 +107,4 @@ llvm = [
 ]
 debug = ["fern", "log", "wasmer-wasi/logging"]
 disable-all-logging = ["wasmer-wasi/disable-all-logging"]
-headless = []
-headless-minimal = ["headless", "disable-all-logging", "wasi", "native", "jit"]
+headless = ["disable-all-logging", "wasi", "native", "jit"]
```

Comment:
I wanted to make a distinction between "headless" (should disable certain features) and "headless-minimal" (compile it to be as small as possible"), because it may be useful to build in `headless` mode without those features

Reply:
The thing we use now `wasmer-headless` binary for it (for creating the minimal headless).
So I thought we already have redundancy with the new binary name (since it can be only generated via xargo), and as such we don't need an extra flag to indicate it. Thoughts?

Reply:
wasmer-headless can be generated without `xargo`, in fact, making `cli` have 2 `bin`s means `wasmer-headless` will be produced every time

Reply:
Via the `Makefile` only the `wasmer` bin would be generated.
I specifically added the `--bin` flags to the `make build-wasmer` commands

Reply:
Okay, I didn't realize that!

Another thing to consider is that since we're 1.0.0, we have to be careful about what the features mean: I don't think it makes sense for `headless` to imply `disable-all-logging`. If we want to only have 1 feature I think we should delete `headless-minimal` and keep `headless = []` as it was. Then we can manually specify the feature set for the minimal build in the Makefile.

I suppose breaking changes in the cli crate is less of an issue because we're not publishing it currently now that I think about it. But even for aesthetic reasons, I don't think we should tie disabling features to headless builds to enabling other features. I think headless wasmer with logging for example is a sensible thing to do.

Reply:
Yeah, I think that's a good tradeoff... can we do those changes on your PR directly?

Reply:
Fixed